src/server/scripts/Spells/spell_dk.cpp

```diff
@@ -660,6 +660,25 @@ class spell_dk_festering_strike : public SpellScript
     }
 };
 
+// 195621 - Frost Fever
+class spell_dk_frost_fever_proc : public AuraScript
+{
+    bool Validate(SpellInfo const* spellInfo) override
+    {
+        return ValidateSpellEffect({ { spellInfo->Id, EFFECT_0 } });
+    }
+
+    bool CheckProc(AuraEffect const* /*aurEff*/, ProcEventInfo const& /*eventInfo*/) const
+    {
+        return roll_chance_i(GetEffect(EFFECT_0)->GetAmount());
```

Comment:
It's always recommendable to pass CalcAmount in case any effect modifies this amount

Reply:
that is for effects which aren't within same aura, theres no need to recalculate for no reason

Reply:
but what if a different aura modifies base value of this aura?

Reply:
thats already taken into account in `GetAmount`